stories/DonutChart.tsx

```diff
@@ -0,0 +1,65 @@
+import * as React from 'react';
+import { storiesOf } from '@storybook/react';
+import { screenshotWrap } from '@gooddata/test-storybook';
+
+import { PieChart } from '../src';
```

Comment:
I think it should be reversed, i.e. importing { DonutChart } here (in DonutChart.tsx) and importing PieChart in PieChart stories below